tests/services/notification.test.js

```diff
@@ -0,0 +1,19 @@
+import { should as importedShould } from 'chai';
+
+import notification from '../../src/services/notification.service';
+
+const should = importedShould();
+
+describe('Notification System', () => {
+  it('Should return undefined for unsupported types', () => {
```

Comment:
```suggestion
  it('should return undefined for unsupported types', () => {
```
Kindly correct in all your files